tests/stdlib/test_time.py

```diff
@@ -106,12 +106,19 @@ def test_clock(self):
 
     #######################################################
     # ctime
-    @expectedFailure
     def test_ctime(self):
         self.assertCodeExecution("""
             import time
-            print(time.ctime())
+            print(time.ctime()[:10], time.ctime()[-4:])
             """)
+    def test_ctime_with_parameter(self):
+        self.assertCodeExecution("""
+            import time
+            print(time.ctime(0))
+            print(time.ctime(1000))
+            now = time.time()
+            print(time.ctime((now - (now % 3600))))
+        """)
```

Comment:
Can you add a test for the case of wrong parameter, like if we give it a string `time.ctime('today')` ?
It should show the same message as CPython (you'll need to do add a type-check for the argument).